### DISCUSSION TITLE
GetProvisioningTemplate causing 403

### DISCUSSION THREAD

**Discussion by thomaspalmie** [pnp/pnpcore#521] [Q&A]
I'm trying to save a site collection template using:
ProvisioningTemplateCreationInformation ptci = new ProvisioningTemplateCreationInformation(ctx.Web);

ptci.FileConnector = new FileSystemConnector(@"c:\temp\template", "");
ptci.PersistBrandingFiles = false;
ptci.ProgressDelegate = delegate (String message, Int32 progress, Int32 total)
{
Console.WriteLine("{0:00}/{1:00} - {2}", progress, total, message);
};

ProvisioningTemplate template = ctx.Web.GetProvisioningTemplate(ptci);

but I'm getting an 403.
I'm authenticating using a tenant admin account:
ctx.Credentials = new SharePointOnlineCredentials(userName, pwd);

I could read the title of the site collection just fine.
Any ideas what's causing this 403?

**Comment by jansenbe** [comment]
@thomaspalmie : please create this as an issue in PnP Framework, this is not related to PnP Core.
